Description: Upstream changes introduced in version 4.6.0-0ubuntu4
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 unity (4.6.0-0ubuntu4) oneiric; urgency=low
 .
   * debian/control:
     - remove the recommends on ubuntuone-control-panel-gtk now that
       ubuntu-desktop recommends it.
     - transition to boost 1.46
     - dep on latest compiz-dev for boost transition
 .
 The person named in the Author field signed this changelog entry.
Author: Didier Roche <didrocks@ubuntu.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- unity-4.6.0.orig/services/CMakeLists.txt
+++ unity-4.6.0/services/CMakeLists.txt
@@ -39,7 +39,7 @@ set(CFLAGS
     "-DGETTEXT_PACKAGE=\"unity\""
     "-DINDICATORDIR=\"${_indicatordir}\""
     "-DINDICATORICONDIR=\"${_iconsdir}\""
-    "-Werror -Wall"
+    "-Wall"
     )
 add_definitions(${CFLAGS})
 
--- unity-4.6.0.orig/plugins/unityshell/src/BGHash.cpp
+++ unity-4.6.0/plugins/unityshell/src/BGHash.cpp
@@ -297,10 +297,10 @@ namespace unity {
     ubus_server_send_message(ubus_server_get_default(),
                              UBUS_BACKGROUND_COLOR_CHANGED,
                              g_variant_new ("(dddd)",
-                                            0.3,
-                                            0.3,
-                                            0.3,
-                                            0.9)
+                                            0.0,
+                                            0.0,
+                                            0.0,
+                                            0.95)
                                             //FIXME - using default colours for now
                                             //_current_color.red,
                                             //_current_color.green,
